app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -163,6 +164,19 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private val syncedTabFeature by lazy {
+        RecentSyncedTabHomeFeature(
+            store = requireComponents.appStore,
+            context = requireContext(),
+            addNewTabUseCase = requireComponents.useCases.tabsUseCases.addTab,
+            navController = findNavController(),
+            lifecycleScope = lifecycleScope,
+            storage = requireComponents.backgroundServices.syncedTabsStorage,
```

Comment:
I'll update it, but I'm curious as to why we'd like to. `Components.syncedTabsStorage` is `lazyMonitored` in case that hook is ever used again, and immediately initializes the underlying `Lazy<RemoteTabsStorage>`. The `syncedTabFeature` is also lazily instantiated in the `HomeFragment`.

Reply:
Additionally, it doesn't look like there's a `Lazy<SyncedTabsStorage>`. Would it be best to add one?